.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "plugin-ceno/pyFreenet"]
 	path = plugin-ceno/pyFreenet
```

Comment:
Apparently this plugin isn't used? It's mentioned somewhere in the docs so I didn't try removing it.